rubicon_ml/intake_rubicon/publish.py

```diff
@@ -181,4 +183,20 @@ def _build_catalog(experiments, visualization):
 
         # append visualization object to end of catalog file
 
+        # vizualization is an MetricListsComparison
+        if isinstance(visualization, MetricListsComparison):
+            appended_visualization_catalog = {
+                "driver": "rubicon_ml_metric_list",
+                "args": {
+                    "column_names": visualization.column_names,
+                    "experiments": visualization.experiments,
```

Comment:
we don't want to log the experiments here. they'll be handled by the experiment source in the catalog like with the other viz sources